integration_tests/e2e/receptionConfirmMove.cy.ts

```diff
@@ -163,16 +163,16 @@ describe.skip('Reception full journey', () => {
     page.form().moveReason().type('Urgent medical appointment')
     page.form().submitButton().click()
 
-    cy.title().should('eq', 'No space available in reception - Digital Prison Services')
+    cy.title().should('eq', `Change Someone's Cell - No space available in reception`)
     cy.get('.govuk-back-link')
       .invoke('attr', 'href')
       .then(href => {
-        expect(href).to.equal('http://localhost:3008/prisoner/G3878UK/reception-move/confirm-reception-move')
+        expect(href).to.include('/prisoner/G3878UK/reception-move/confirm-reception-move')
       })
     cy.get('[data-test="location-details-link"]')
       .invoke('attr', 'href')
       .then(href => {
-        expect(href).to.equal('/prisoner/G3878UK/location-details')
+        expect(href).to.equal('http://localhost:3101/prisoner/G3878UK/location-details')
```

Comment:
This should use config'd host rather than hard coded?

Reply:
It's the value from the `feature.env` file so it is a predictable value. I can change it to get it from the config but it's done this way in quite a few of the tests so they would need to be updated too.

Reply:
Some say it's better to test for explicit values rather than references, others say the opposite, I'm not sure it makes a huge difference...